code/src/mlx90640_i2c.cpp

```diff
@@ -0,0 +1,21 @@
+#include <mlx90640_i2c.hpp>
+
+namespace r2d2::thermal_camera {
+    mlx90640_i2c_c::mlx90640_i2c_c(i2c::i2c_bus_c &bus, const uint8_t address)
+        : bus(bus), address(address) {
+    }
+
+    uint16_t
```

Comment:
this looks pretty strange a uint16_t on a single line.

Reply:
I applied `.clang-format` on the file, but nothing changed. I think that's how it should be formatted. I don't think it looks that good myself, but oh well.

Reply:
you can format it manualy as:

```
    uint16_t mlx90640_i2c_c::read_register(const uint16_t internal_address)
        const {
        // something
    }
```